types/webextension-polyfill/namespaces/storage.d.ts

```diff
@@ -39,7 +34,7 @@ export namespace Storage {
          * to get the entire contents of storage.
          * @returns Callback with storage items, or on failure (in which case $(ref:runtime.lastError) will be set).
          */
-        get(keys?: null | string | string[] | Record<string, any>): Promise<Record<string, any>>;
+        get(keys?: null | string | string[] | Record<string, unknown>): Promise<Record<string, unknown>>;
```

Comment:
This seems like another change from `any` to `unknown` in an output position. Is the record type in the return type meant to be the same type as the record type in the parameter type? If so, maybe there should be a type parameter added here.

Reply:
Same reason: only the caller knows what type it's supposed to be.

The returned record will contain keys as specified in the first parameter and possibly default values when the first parameter is an object. But default values are not necessarily the only type that can be stored here, so it's not a 100% match even if I was to create complex code to generate these kinds of overload.

Reply:
I see. If this is the kind of thing that only the caller is supposed to know, then I'm inclined to think we should keep those types as `any`s. There's not much of a point to using `unknown` if we expect the caller to have to cast the result to some specific type.

Reply:
The point would be, that a required explicit cast makes it harder to do something wrong, as you have less implicit `any`s.